lib/Screen/HomeScreen.dart

```diff
@@ -0,0 +1,168 @@
+                      child: const Text(
+                        "Please enter your email address",
+                        style: TextStyle(
+                            color: Colors.white,
+                            fontSize: 20,
+                            fontWeight: FontWeight.w500),
+                      ),
+                    ),
+                    Container(
+                      padding: const EdgeInsets.fromLTRB(30, 20, 30, 20),
+                      child: TextField(
+                        style: const TextStyle(
+                            color: Colors.white,
+                            fontWeight: FontWeight.w500,
+                            fontSize: 20),
+                        decoration: const InputDecoration(
+                          isCollapsed: true,
+                          contentPadding: EdgeInsets.all(10),
+                          filled: true,
+                          fillColor: Color.fromRGBO(255, 255, 255, 0.4),
+                          enabledBorder: OutlineInputBorder(
+                              borderRadius:
+                                  BorderRadius.all(Radius.circular(5)),
+                              borderSide:
+                                  BorderSide(color: Colors.white, width: 2.0)),
+                          focusedBorder: OutlineInputBorder(
+                            borderRadius: BorderRadius.all(Radius.circular(5)),
+                            borderSide:
+                                BorderSide(color: Colors.white, width: 2.0),
+                          ),
+                          hintText: "Email address",
+                          hintStyle: TextStyle(
+                              color: Colors.white,
+                              fontWeight: FontWeight.w500,
+                              fontSize: 20),
+                        ),
+                        onChanged: (value) {
+                          setState(() {
+                            if (value.length == 0) {
+                              _isButtonDisable = false;
+                            } else {
+                              _isButtonDisable = true;
+                            }
+                          });
+                          setState(() {
+                            emailValue = value;
+                          });
+                        },
+                      ),
+                    )
+                  ],
+                )),
+                Container(
+                  padding: const EdgeInsets.fromLTRB(30, 0, 30, 0),
+                  // color:Colors.amberAccent,
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.stretch,
+                    children: <Widget>[
+                      ElevatedButton(
+                          style: ButtonStyle(
+                            backgroundColor: !_isButtonDisable
+                                ? MaterialStateProperty.all<Color>(
```

Comment:
ElevatedButtonTheme is already defined in ThemeData, you don't need to override it here unless you wanna use different theme